lib/rubocop/cop/minitest/global_expectations.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # This Cop checks for deprecated global expectations
+      # and autocorrects them to use expect format
```

Comment:
Can you add a period at the end of the sentence?

```suggestion
      # and autocorrects them to use expect format.
```

Reply:
@koic Added.